src/controllers/executionPlanWebviewController.ts

```diff
@@ -104,7 +109,7 @@ export class ExecutionPlanWebviewController extends ReactWebviewPanelController<
 
             if (saveUri) {
                 // Write the content to the new file
-                await vscode.workspace.fs.writeFile(
+                vscode.workspace.fs.writeFile(
```

Comment:
if `writeFile` is async and we don't want to await it, then we should explicitly use the `void` keyword.